File: Ad-Hoc/2293/2293.c
#include <stdio.h>

int main(){
    int n,m, maior = 0, aux;
    scanf("%d %d", &n, &m);
    int linha[n], coluna[m];
    for (int x = 0; x<n; x++){
        linha[x] = 0;
    }
    for (int y = 0; y<m; y++){
        coluna[y] = 0;
    }
    for (int i = 0; i<n; i++){
        for (int j = 0; j<m; j++){
            scanf("%d", &aux);
            linha[i] += aux;
            coluna[j] += aux;
        }
        

    }
    for (int i = 0; i<n; i++){
        if (linha[i]>maior){
            maior = linha[i];
        }
    }
    for (int j = 0; j<m; j++){
        if (coluna[j]>maior){
            maior = coluna[j];
        }
    }
    printf("%d\n", maior);
    return 0;
}

File: Ad-Hoc/2293/2293.py
N,M = map(int, input().split())
if N>=1 and N<=100 and M>=1 and M<=100:
    coluna = [0] * M
    linha = [0] * N
    maior = 0
    for x in range(N):
        lista = input().split()
        for y in range(len(lista)):
            linha[x] += int(lista[y])
            coluna[y] += int(lista[y])
    maior = max(coluna) if max(coluna)>=max(linha) else max(linha)
    print(maior)